### PR TITLE
Implemented demo mode using filter_complex

### PR DESCRIPTION
**Pull request by zerkms**
Now it does all the combining slides + speaker + demos in a single pass, hooray!

PS: the wiki also must be fixed to add the new command, and I believe the old `h264_overlay` would be deprecated as soon as the `h264_overlay_demo` is confirmed to be working by someone else.

### CODE REVIEW

**Comment by cykl** [comment]
Thanks for the patch ! (and sorry for the delay)

I am testing your patch, will then try to understand most of ffmepg oddities and merge it.

Is there any reason to not replace the current `h264_overlay` output  with your implementation ?

File: infoqscraper/convert.py [hunk 1]
```diff
@@ -210,18 +211,107 @@ def _ffmpeg_h264_overlay(self, audio, frame_pattern):
...
+
+        cmd += inputs
+
+        cmd += [
+            "-filter_complex_script", filter_script_path,
+            "-map", "[v]", "-map", "[a]",
+            "-r", str(fps),
+            "-acodec", "libmp3lame", "-ab", "92k",
+            "-vcodec", "libx264", "-profile:v", "baseline", "-preset", "fast", "-level", "3.0", "-crf", "28",
+            "-n",
```
Comment (cykl) [inline infoqscraper/convert.py]: Is there a reason to not use `"-y" if self.overwrite else "-n",` like in other outputs ?

**Comment by cykl** [comment]
LGTM

I locally amended your PR to replace the old `h264_overlay` by your better implementation and restore the support of `--overwrite`. 

Let me know if you are OK with these two changes and I will merge it & push a new release

File: infoqscraper/convert.py (hunk 1 above)
Comment (zerkms) [inline infoqscraper/convert.py]: @cykl there is no at all

**Comment by zerkms** [comment]
@cykl it is OK surely :-)

**Pull request merged**